test/acceptance/behave/README.md

```diff
@@ -120,11 +119,6 @@ file in the same directory that contains the “steps” directory.
        * UPDATE_PROPERTIES_JSON: determine whether you are create/update `properties.json` automatically or manually (used to jenkins).
            - true: read external file (the file is mandatory) in `PATH_TO_SETTING_FOLDER` (with ssh commands) and execute these automatically (used to create `properties.json`) 
            - false: it does nothing and the creation of `properties.json` will be by user in jenkins console.
-       * CB_RUNNING_MODE: is used to determine how is compiled ContextBroker.
```

Comment:
behave test has not been used for year. However, the removal of the CB_RUNNING_MODE selectors seems to be pretty straightforward, so I did that.

NTC (informative)